xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -1392,7 +1392,25 @@ bool CDVDVideoCodecAndroidMediaCodec::ConfigureMediaCodec(void)
   if (CJNIBase::GetSDKVersion() >= 24)
   {
     if (m_hints.colorRange != AVCOL_RANGE_UNSPECIFIED)
```

Comment:
Why don't you handle this case in the switch case statement below?

Reply:
For consistency with the existing code I followed the handling logic used with the other video flags and it seemed to me to be fine. If the value is "Unspecified" we avoid directly doing the case condition checks.

Reply:
If you put the "unspecified" as first case there should be no overhead in the generated code, just one comparism. But with the code as it is now, if the value is not "unspecified" you will do at least two comparisons, which IS overhead.

Reply:
I know, as I said I wanted to use the same style as the similar code in the method. I'll change it as you suggest if we see it better

Reply:
Thanks.